foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/URIEndpointObject.java

```diff
@@ -47,6 +53,10 @@ public URIEndpointObject(String endpoint) {
     setPort(uri.getPort());
     querys = splitQuery(uri);
     sslEnabled = Boolean.parseBoolean(getFirst(SSL_ENABLED_KEY));
+    String httpversion = getFirst(HTTP2_ENABLED_KEY);
+    if (httpversion != null && httpversion.compareTo(HTTP2) == 0) {
```

Comment:
http2Enabled  = HTTP2.equals(httpVersion)

Reply:
Fixed and comitted